projects/plugins/jetpack/extensions/blocks/like/like.php

```diff
@@ -87,7 +87,7 @@ function render_block( $attr, $content, $block ) {
 		$bloginfo = get_blog_details( (int) $blog_id );
 		$domain   = $bloginfo->domain;
 		$version  = '20231201';
-		$src      = sprintf( '//widgets.wp.com/likes/index.html?ver=%1$d#blog_id=%2$d&amp;post_id=%3$d&amp;origin=%4$s&amp;obj_id=%2$d-%3$d-%5$s%6$s', $version, $blog_id, $post_id, $domain, $uniqid, $new_layout );
+		$src      = sprintf( '//widgets.wp.com/likes/index.html?ver=%1$d#blog_id=%2$d&amp;post_id=%3$d&amp;origin=%4$s&amp;obj_id=%2$d-%3$d-%5$s%6$s&block=1', $version, $blog_id, $post_id, $domain, $uniqid, $new_layout );
```

Comment:
I think the `&` should be encoded like `&amp;` as the others in the same URL, and be placed before the `obj_id`. All the other parameters are reflected in the `obj_id`, so I think this one should follow the same approach and be passed as a boolean variable to the `sprintf`.